### ISSUE TITLE
Implicit default selectors to simplify :select usage

### ISSUE THREAD

**Issue by glts** [jgpc42/jmh-clojure#2]
The selector mechanism is very powerful and general. However, the most
frequent selector use in my experience, ‘run just benchmarks x and y’,
is not convenient to set up: I have to define an explicit selector
predicate for each benchmark.

```
:benchmarks
[{:name :foo, :fn my.app/foo}
 {:name :bar, :fn my.app/bar}
 {:name :baz, :fn my.app/baz}
 ,,,
 ]

:selectors
{:foo (comp #{:foo} :name)
 :bar (comp #{:bar} :name)
 :baz (comp #{:baz} :name)
 ,,,
 }
```

Unless I am overlooking something obvious each benchmark needs a
selector mirroring it. That’s boilerplate configuration.

Would it make sense to provide an implicit default selector for each
benchmark, perhaps based on `:name`? (Enabling `:select [:foo]`
directly, where `:foo` is the name of the benchmark.)

I haven’t thought it through, just bringing this to your attention …
last missing piece for me before I am 100% happy with jmh-clojure,
promise.

**Comment by jgpc42** [comment]
My original design for `:selectors` was based on Leiningen's `:test-selectors`, but implicit selectors is a worthwhile improvement. And `:name` is the best candidate: I was actually just writing boilerplate selectors like you describe yesterday!

Implemented with 7d96f2c. Deployed 0.1.5. I'm also deploying lein-jmh 0.2.1.

And you should feel free to submit future feature requests. Thanks again!

**Comment by glts** [comment]
Amazing. Thank you.
